### PR TITLE
Merge all changes in receipt, booking modal screen

### PR DESCRIPTION
**Pull request by gabrieldiazmorro**
Merge all changes in receipt, booking modal screen

### CODE REVIEW

**Review by gabrieldiazmorro — COMMENTED** [review]
LGTM

File: react_native/src/components/CourseMasterModalComponent.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    let [value, setState] = useState(true);
+    return () => setState(!value);
+}
+
+function CourseMasterModalComponent(props) {
+
+    const {bookingData, updateBookingData} = useContext(BookingContext);
+    let forceUpdate = callForceUpdate();
+    const handleAddMaster = () => {
+        axios.post("http://192.168.0.14:8080/tuter/masters",
```
Comment (bermed28) [inline react_native/src/components/CourseMasterModalComponent.js:21]: @gabrieldiazmorro change these

File: react_native/src/components/CourseMasterModalComponent.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                updateBookingData.clear();
+                props.navigation.navigate("Home");
+            }
+        ).catch((error) => {Alert.alert(`Cannot Master ${props.master.course_code}`, error.message)});
+
+    }
+
+    const handleRemoveMaster = () => {
+        console.log(props.navigation)
+        axios.delete(`http://192.168.0.14:8080/tuter/masters/${bookingData.user.user_id}`,
```
Comment (bermed28) [inline react_native/src/components/CourseMasterModalComponent.js:35]: @gabrieldiazmorro also here

File: react_native/src/components/IncrementDecrementComponent.js [hunk 3]
```diff
@@ -6,7 +6,11 @@ function IncrementDecrementComponent(props){
 
     return(
         <View style={{flex: 1, flexDirection: "row", alignItems:"center", justifyContent: "space-evenly"}}>
-            <TouchableOpacity onPress={props.onChangeIncrement}>
+        {/*<View style={{flex: 1, flexDirection: "row", alignItems:"center", justifyContent: "flex-end"}}>*/}
```
Comment (bermed28) [inline react_native/src/components/IncrementDecrementComponent.js:9]: @gabrieldiazmorro delete this commented code

File: react_native/src/components/SessionBookingModalComponent.js [hunk 4]
```diff
@@ -266,91 +295,136 @@ function SessionBookingModalComponent(props) {
...
                         borderColor: "#000000",
                         padding: 5,
                         paddingRight: 5,
                         shadowRadius: 10,
                         shadowOffset: {width: 0, height: 3},
-                        shadowColor: "rgba(0,0,0,0.75)"}}>
-                        <Text>Location</Text>
+                        shadowColor: "rgba(0,0,0,0.75)"
+                    }}>
+                        {/*<Text style={styles.text_footer}>Location</Text>*/}
```
Comment (bermed28) [inline react_native/src/components/SessionBookingModalComponent.js:398]: @gabrieldiazmorro delete the comments in this file

**Review by bermed28 — CHANGES_REQUESTED** [review]
Address these comments please!

**Review by bermed28 — APPROVED** [review]
LGTM

**Review by Chris36021 — APPROVED** [review]
lgtm

**Pull request merged**